src/Schema/Values/ContextModule.ts

```diff
@@ -66,6 +66,11 @@ export enum ContextModule {
   moreSeriesByThisArtist = "moreSeriesByThisArtist",
   myCollectionArtwork = "myCollectionArtwork",
   myCollectionHome = "myCollectionHome",
+  myCollectionArtworkAuctionResults = "myCollectionArtworkAuctionResults",
+  myCollectionArtworkHeader = "myCollectionArtworkHeader",
+  myCollectionArtworkMarketInsights = "myCollectionArtworkMarketInsights",
+  myCollectionArtworkRelatedArticles = "myCollectionArtworkRelatedArticles",
+  myCollectionArtworkSell = "myCollectionArtworkSell",
```

Comment:
these are great. thinking though, do they need to be `myCollectionArtwork` specific if the screen owner specifies that much already?

Reply:
we can do `marketInsights`, `auctionResults` etc.

Reply:
Updated, it looks like we have generic modules for most of these already just adding the marketInsights and artworkMetadata:

Existing contextModules:
https://github.com/artsy/cohesion/blob/135c07c4b9d9507424dbd09bf081fc15178b564f/src/Schema/Values/ContextModule.ts#L95
https://github.com/artsy/cohesion/blob/135c07c4b9d9507424dbd09bf081fc15178b564f/src/Schema/Values/ContextModule.ts#L29
https://github.com/artsy/cohesion/blob/135c07c4b9d9507424dbd09bf081fc15178b564f/src/Schema/Values/ContextModule.ts#L100